packages/functions/src/serializer.ts

```diff
@@ -53,27 +53,27 @@ export class Serializer {
       return data.map(x => this.encode(x));
     }
     if (typeof data === 'function' || typeof data === 'object') {
-      return mapValues(data, x => this.encode(x));
+      return mapValues(data as object, x => this.encode(x));
     }
     // If we got this far, the data is not encodable.
     throw new Error('Data cannot be encoded in JSON: ' + data);
   }
 
   // Takes data that's been encoded in a JSON-friendly form and returns a form
   // with richer datatypes, such as Dates, etc.
-  decode(json: any): any {
-    if (json === null) {
+  decode(json: unknown): unknown {
```

Comment:
This should be `decode(json: {} | null): {} | null {`. The callers already seem fine with that. Passing in `undefined` would've thrown anyway.

Reply:
Good point. updated.